packages/react-charts/src/components/ChartBullet/__snapshots__/ChartBullet.test.tsx.snap

```diff
@@ -34,24 +34,24 @@ exports[`ChartBulletQualitativeRange 1`] = `
             shape-rendering="auto"
             style="stroke: var(--pf-chart-axis--tick--stroke--Color, #d2d2d2); fill: var(--pf-chart-axis--tick--Fill, transparent); size: 5px; stroke-linecap: round; stroke-linejoin: round; stroke-width: 1;"
             vector-effect="non-scaling-stroke"
-            x1="50"
-            x2="50"
+            x1="225"
```

Comment:
What do these values mean and why are they changed? Curious as I'm not seeing a data change.

Reply:
Looks like it might have something to do with a label being on the left of the chart (so like an offset) based on values I'm seeing while inspecting examples.. not sure why this snapshot was changed, but as the examples look good i'm not too worried

Reply:
These changes look related to the tick axis label. 

Can't really pinpoint the exact change between v36.2.1 and v36.6.7. However, the bullet chart examples look good to me.

The change log can be found here
https://github.com/FormidableLabs/victory/blob/main/CHANGELOG.md